docs/dml-of-carbondata.md

```diff
@@ -35,10 +35,14 @@ CarbonData DML statements are documented here,which includes:
   This command is used to load csv files to carbondata, OPTIONS are not mandatory for data loading process. 
 
   ```
-  LOAD DATA [LOCAL] INPATH 'folder_path' 
+  LOAD DATA INPATH 'folder_path'
   INTO TABLE [db_name.]table_name 
   OPTIONS(property_name=property_value, ...)
   ```
+  **NOTE**:
+   - If run the sql on cluster mode, please upload all input files to HDFS or S3 and so on.
+   - If run the sql on local mode with HDFS, please upload all input files to HDFS.
+   - If run the sql on local mode with local file system, it just supports to read input files from local file system.
```

Comment:
I guess no need to mention all these points, just ask the user to provide the path along with scheme like file:// or hdfs:// or s3:// etc.

Reply:
Done, please review, thanks. @ravipesala